src/PhpBrew/Build.php

```diff
@@ -83,6 +90,35 @@ public function __construct($version, $name = null, $installPrefix = null)
         $this->osRelease = php_uname('r');
     }
 
+    public function getPkgConfigPaths()
+    {
+        return array_keys($this->pkgConfigPaths);
+    }
+
+    public function addPkgConfigPath($path)
+    {
+        $this->pkgConfigPaths[$path] = true;
+    }
+
+    public function putPkgConfigPathsEnv()
+    {
+        $paths = $this->getPkgConfigPaths();
+        $pathCount = count($paths);
+
+        if ($pathCount == 0) {
+            return false;
+        } else {
+            $currentPkgConfigPath = getenv('PKG_CONFIG_PATH');
+            if ($currentPkgConfigPath !== '') {
```

Comment:
It can also be `false`. Please take that into account:
```sh
php -r "var_dump(getenv('BLAH'));"
bool(false)
```

Reply:
I'll check it.